packages/calypso-storybook/src/index.js

```diff
@@ -41,12 +41,13 @@ module.exports = function storybookDefaultConfig( {
 		addons: [
 			'@storybook/addon-actions',
 			'@storybook/addon-controls',
+			'@storybook/addon-docs',
 			'@storybook/addon-viewport',
 			'@storybook/preset-scss',
 		],
 		typescript: {
 			check: false,
-			reactDocgen: false,
+			reactDocgen: 'react-docgen-typescript',
```

Comment:
If you relaunch Storybook after setting this back to `false`, you will see that the Controls aren't generated.